src/js/DrawStyleController.js

```diff
@@ -191,6 +229,15 @@ goog.require('ga_styles_service');
     $scope.options.icon = options.icons[0];
     $scope.options.iconColor = options.colors[5];
     $scope.options.iconSize = options.iconSizes[2];
+    $scope.options.iconCategory = options.iconCategories[0];
+
+    $scope.options.iconCategories.forEach(function(category) {
+      if (category.type === 'img') {
+        category.icons = getImgsByCategory('babs');
+      } else {
+        category.icons = $scope.options.icons;
+      }
+    });
```

Comment:
After the iconCategories is defined set the option.icons to $scope.options.iconCategory.icons